modules/preview-react/multi-select/lib/MultiSelectInput.tsx

```diff
@@ -28,12 +30,14 @@ export const multiSelectStencil = createStencil({
     minHeight: system.space.x10,
     transition: '0.2s box-shadow, 0.2s border-color',
     margin: 0, // Fix Safari
+    [textInputStencil.vars.width]: '100%',
 
     '&:hover, &.hover': {
       borderColor: system.color.border.input.strong,
     },
 
-    '&:focus-within': {
+    '&:has(:focus-visible:not([disabled])), &.focus': {
+      // '&:has(:focus-visible)': {
```

Comment:
```suggestion
```

Reply:
That reminds me. I don't like the selector.